Userland/Services/WebContent/ConnectionFromClient.cpp

```diff
@@ -398,6 +398,11 @@ void ConnectionFromClient::debug_request(u64 page_id, ByteString const& request,
         }
         return;
     }
+
+    if (request == "do-not-track") {
+        Web::ResourceLoader::the().set_enable_do_not_track(argument == "on");
+        return;
+    }
```

Comment:
I would say this isn't a "debug request", but rather a first-class feature that deserves its own IPC

Reply:
Fair enough, I'll work on that later :)

Reply:
I've made this change :)